README.md

```diff
@@ -767,6 +904,9 @@ rules.when().item('HallLight').receivedUpdate().then().copyState().fromItem('Bed
 ```
 
 ### Event Object
+
+**NOTE**: This section is only valid for File Based Rules. If you are using MainUI with jsscripting

```

Comment:
This seems incomplete, I think you forgot something here.

I would write:

> **NOTE**: This section is only valid for file-based scripts/rules. If you use JS Scripting in MainUI, please refer to ...

Reply:
https://github.com/openhab/openhab-js/pull/110/commits/a49c8d9f526685048d7f64f6a1017ae48ef1e40c